### PR TITLE
PauseMenu

### PR DESCRIPTION
**Pull request by SafuanA**
Review

### CODE REVIEW

File: Assets/Scripts/UI/PauseMenu.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    {
+        pauseMenu.SetActive(true);
+        Time.timeScale = 0f;
+        isGamePaused = true;
+    }
+
+    public void LoadMenu()
+    {
+        SceneManager.LoadScene("Menu");
+        Time.timeScale = 1f;
```
Comment (zicons) [inline Assets/Scripts/UI/PauseMenu.cs:46]: why do you set the timescale here to 1? if we are in the menu..

File: Assets/Scripts/UI/PauseMenu.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+public class PauseMenu : MonoBehaviour
+{
+    public static bool isGamePaused = false;
+
+    [SerializeField] public GameObject pauseMenu; 
+    
+    void Start () 
+    {
+        Debug.Log("Hello world!");
```
Comment (zicons) [inline Assets/Scripts/UI/PauseMenu.cs:12]: do you need this start function? i think you can remove it

**Review by zicons — COMMENTED** [review]
You have to merge dev into your branch... you are way behind.. you have all the old assets.

**Review by abcdam — CHANGES_REQUESTED** [review]
debug line can be removed, otherwise it looks ok

File: Assets/Scripts/UI/PauseMenu.cs (hunk 1 above)
Comment (SafuanA) [reply Assets/Scripts/UI/PauseMenu.cs:42]: The timeScale is set to 0 when the pause menu is clicked. When you go back to the Menu the timescale should be set to normal

**Review by SomethingWittyWithATechAspect — APPROVED** [review]

**Pull request merged**